index.js

```diff
@@ -445,16 +445,26 @@ class GithubScm extends Scm {
             scmUri: config.scmUri,
             token: config.token
         });
-        const repo = await this.breaker.runCommand({
-            action: 'get',
-            token: config.token,
-            params: {
-                owner: scmInfo.owner,
-                repo: scmInfo.repo
+
+        try {
+            const repo = await this.breaker.runCommand({
+                action: 'get',
+                token: config.token,
+                params: {
+                    owner: scmInfo.owner,
+                    repo: scmInfo.repo
+                }
+            });
+
+            return repo.data.permissions;
+        } catch (err) {
+            // Suspended user
+            if (err.code === 403) {
+                return { admin: false, push: false, pull: false };
```

Comment:
I think it would be good to not lose the "Account suspended" information. We can still mask the error and normalize output, but can we also log out something like `User's account suspended for ${owner}/${repo}, removing from pipeline admins.`?

Reply:
I added logging in this module.
In addition I think we should add logging code appropriately in another module too.
e.g. https://github.com/screwdriver-cd/models/blob/master/lib/pipeline.js#L744